### DISCUSSION TITLE
Estrutura de projeto

### DISCUSSION THREAD

**Discussion by BrunoBafilli** [balta-io/2801#19] [Perguntas e Respostas] [answered]
Eu gostaria de saber por que no dot.net6 nao contem os seguintes itens

Using system
NameSpace
Class Program
Static void main

Quando eu crio o projeto, para mim aparece a penas  o console.log("hellow world")

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @BrunoBafilli como vai?

O .NET 6 ainda não foi lançado, peço por favor que siga o curso com o .NET 5 que é a versão mais atual.

Sobre os pontos questionados, no .NET 6 não será mais obrigatório o uso explicito dos itens mencionados, porém, se você copiar e colar o código do curso (.NET 5) em um novo projeto (.NET 6) ele vai funcionar, tem total retrocompatibilidade.

Nos cursos novos de EF Core e ASP.NET 6 (Previstos para Novembro) já incluem todas estas mudanças 💜
